client/my-sites/marketplace/pages/marketplace-plugin-upload-status/index.tsx

```diff
@@ -256,14 +259,31 @@ const MarketplacePluginInstall = ( {
 			return (
 				<EmptyContent
 					illustration="/calypso/images/illustrations/error.svg"
-					title={ translate(
+					title={ null }
+					line={ translate(
 						'This URL should not be accessed directly. Please click the Install button on the plugin page.'
 					) }
 					action={ translate( 'Go to the plugin page' ) }
 					actionURL={ `/plugins/${ productSlug }/${ selectedSite?.slug }` }
 				/>
 			);
 		}
+		if ( pluginExists ) {
+			return (
+				<EmptyContent
+					illustration="/calypso/images/illustrations/error.svg"
+					title={ null }
+					line={ translate(
+						'This plugin already exists in your site. If you want to upgrade/ downgrade it please continue by uploading again from WP Admin.'
```

Comment:
Shouldn't we use `upgrade/downgrade` instead of `upgrade/ downgrade`?

```suggestion
						'This plugin already exists in your site. If you want to upgrade/downgrade it please continue by uploading again from WP Admin.'
```

Reply:
I am always adding a space after punctuation marks but I don't have a strong opinion here. @emmnyc85 any preference here?

Reply:
'This plugin already exists on your site. If you want to upgrade or downgrade the plugin, please continue by uploading the plugin again from WP Admin.'